lib/Demangling/OldDemangler.cpp

```diff
@@ -2252,6 +2252,9 @@ class OldDemangler {
     if (Mangled.nextIf('H'))
       addImplFunctionAttribute(type, "@async");
 
+    if (Mangled.nextIf('T'))
+      addImplFunctionAttribute(type, "sending-result");
```

Comment:
I don't think we really need to care about the old demangler for impl-function-type. If we do care, though, it's notable that this is producing a different node from the new demangler.